buildconfig/pygame-stubs/math.pyi

```diff
@@ -0,0 +1,84 @@
+from typing import Optional, Union, Tuple, List, overload
+class VectorElementwiseProxy: ...
+
+def enable_swizzling() -> None: ...
+def disable_swizzling() -> None: ...
+
+class Vector2:
+    x: float
+    y: float
+    def __init__(self, x: Optional[Union[float, Vector2, Tuple[float, float], List[float]]]=0, y: Optional[float]=0) -> None: ...
+    def dot(self, other: Vector2) -> float: ...
+    def cross(self, other: Vector2) -> Vector2: ...
+    def magnitude(self) -> float: ...
+    def magnitude_squared(self) -> float: ...
+    def length(self) -> float: ...
+    def length_squared(self) -> float: ...
+    def normalize(self) -> Vector2: ...
+    def normalize_ip(self) -> None: ...
+    def is_normalized(self) -> bool: ...
+    def scale_to_length(self, value: float) -> None: ...
+    def reflect(self, other: Vector2) -> Vector2: ...
+    def reflect_ip(self, other: Vector2) -> None: ...
+    def distance_to(self, other: Vector2) -> float: ...
+    def distance_squared_to(self, other: Vector2) -> float: ...
+    def lerp(self, other: Vector2, value: float) -> Vector2: ...
+    def slerp(self, other: Vector2, value: float) -> Vector2: ...
+    def elementwise(self) -> VectorElementwiseProxy: ...
+    def rotate(self, angle: float) -> Vector2: ...
+    def rotate_rad(self, angle: float) -> Vector2: ...
+    def rotate_ip(self, angle: float) -> None: ...
+    def rotate_ip_rad(self, angle: float) -> None: ...
+    def angle_to(self, other: Vector2) -> float: ...
+    def as_polar(self) -> Tuple[float, float]: ...
+    def from_polar(self, polar_value: Union[List[float], Tuple[float, float]]) -> None: ...
+    def update(self, first: Optional[Union[float, Vector2, Tuple[float, float], List[float]]]=0, second: Optional[float]=0) -> None: ...
```

Comment:
vector.c@vector2_update has keyword arguments of `x, y` not `first` and `second`.

Reply:
Done